lib/resource-mapper.js

```diff
@@ -68,6 +68,12 @@ class ResourceMapper {
     if (path.startsWith(this._rootPath)) {
```

Comment:
I wonder if we should be concerned about backslashes in paths (Windows) at the start of the function. There is another line in this file dealing with backslashes.

Reply:
That's taken care of later in the file

Reply:
Yes, but what if this method is called with the Windows file path `\bla\example.org\foo\bar`?